Cargo.lock

```diff
@@ -2133,11 +2172,11 @@ dependencies = [
  "percent-encoding",
  "pin-project",
  "prost",
- "rustls-native-certs",
+ "rustls-native-certs 0.7.0",
  "rustls-pemfile 2.1.2",
  "rustls-pki-types",
  "tokio",
- "tokio-rustls",
+ "tokio-rustls 0.25.0",
```

Comment:
We're gonna end up with two versions of tokio-rustls until we can upgrade reqwest. This is less than fun, but alright.